### ISSUE TITLE
`REDACT_EMAIL` and `REDACT_IP` can't be disabled

### ISSUE THREAD

**Issue by kiddom-kq** [DataDog/datadog-serverless-functions#499]
**Describe what happened:**

Looking here:
https://github.com/DataDog/datadog-serverless-functions/blob/ff95db6e373597a3bae1cac021a82cd509b48e80/aws/logs_monitoring/template.yaml#L492

It appears as though the `REDACT_EMAIL` and `REDACT_IP` env-vars can be set with a `true` or `false` value to enable or disable the regex rules. The code, however, does not check the value of these environment variables.

The email/IP regexes are created here:
https://github.com/DataDog/datadog-serverless-functions/blob/ff95db6e373597a3bae1cac021a82cd509b48e80/aws/logs_monitoring/settings.py#L161

And then passed directly into the scrubber here:
https://github.com/DataDog/datadog-serverless-functions/blob/118e35a3a162f9fbbc2f98103040eb3aedf69805/aws/logs_monitoring/logs.py#L59

**Describe what you expected:**

I would expect a call to [`get_env_var()` ](https://github.com/DataDog/datadog-serverless-functions/blob/ff95db6e373597a3bae1cac021a82cd509b48e80/aws/logs_monitoring/settings.py#L16) prior to the [`ScrubbingRuleConfig(object)`](https://github.com/DataDog/datadog-serverless-functions/blob/ff95db6e373597a3bae1cac021a82cd509b48e80/aws/logs_monitoring/settings.py#L152) object creation in the [`SCRUBBING_RULE_CONFIGS`](https://github.com/DataDog/datadog-serverless-functions/blob/ff95db6e373597a3bae1cac021a82cd509b48e80/aws/logs_monitoring/settings.py#L161) list.

**Steps to reproduce the issue:**

Deploy the lambda function, making sure to use an environment var config:

```
DD_USE_TCP | false
REDACT_EMAIL | false
```

And observe that cloudtrail logs are scrubbed... despite the env-vars set to disable the feature

**Comment by DarcyRaynerDD** [comment]
@kiddom-kq Thanks for the report. I've checked the code,  it looks like the code does check for the presence of the environment [variable](https://github.com/DataDog/datadog-serverless-functions/blob/118e35a3a162f9fbbc2f98103040eb3aedf69805/aws/logs_monitoring/logs.py#L181), but not whether it's true or false, (setting it will enable the scrubbing rule regardless). I've added a ticket in our backlog to update the behaviour.
